python/ray/_private/runtime_env.py

```
@@ -40,11 +40,17 @@ class RuntimeEnvDict:
             modules to add to the `sys.path`.
             Examples:
                 ["/path/to/other_module", "/other_path/local_project.zip"]
+        pip (str): A string containing the contents of a pip requirements.txt
+            file.  This will be dynamically installed in the runtime env.
         conda (dict | str): Either the conda YAML config or the name of a
             local conda env (e.g., "pytorch_p36"). The Ray dependency will be
             automatically injected into the conda env to ensure compatibility
             with the cluster Ray. The conda name may be mangled automatically
             to avoid conflicts between runtime envs.
+            This field overrides the pip field.  To use pip with conda, please
+            specify your pip dependencies within the conda YAML config:
+            https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-e
+            nvironments.html#create-env-file-manually
```

Instead of overriding can we throw an error telling people about this behavior?